src/pages/download/_autoDownload.ts

```diff
@@ -0,0 +1,13 @@
+export function readyForAutoDownload(fn: () => void): void {
+  if (document.readyState != "loading") {
+    fn();
+  } else {
+    document.addEventListener("DOMContentLoaded", fn);
+  }
+}
+
+export function mainForAutoDownload(): void {
+  generateZIP((window as any).targetDeviceId);
```

Comment:
Do you expect to fix these types in this PR? or later?

If fix type in this PR, please avoid to use any; You might be interested in https://www.typescriptlang.org/tsconfig/#lib

Otherwise, add TODO comment?

Reply:
added TODO